#ifndef MSG_CONSTANT_H
#define MSG_CONSTANT_H
// ********  API  ******
// success message code
#define MSG_CODE_SUCCESS 100

// error message code
#define MSG_CODE_ERROR 200
#define MSG_CODE_INVALID_MSG_PARAM 201
#define MSG_CODE_INVALID_TOKEN 202

// ********  TCP  ******
// client operation type
#define TCP_MSG_OP_TYPE_SEND_MSG 1


// msg type
#define TCP_MSG_SINGLE_CHAT_TYPE 1
#define TCP_MSG_GROUP_CHAT_TYPE 2


// msg success code
#define TCP_MSG_CODE_SUCCESS 100

// msg error code
#define TCP_MSG_CODE_ERROR 200


#endif